src/herder/HerderImpl.cpp

```diff
@@ -1072,7 +1072,8 @@ static Hash
 getQmapHash(QuorumTracker::QuorumMap const& qmap)
 {
     std::unique_ptr<SHA256> hasher = SHA256::create();
-    for (auto const& pair : qmap)
+    std::map<NodeID, SCPQuorumSetPtr> ordered_map(qmap.begin(), qmap.end());
```

Comment:
maybe use a set instead, something like:
```
    std::transform(m.begin(), m.end(), std::inserter(s, s.begin()), [](auto const& pair) { return pair.first; });
```

Reply:
I don't understand .. we need the values of the map too. What would we use the set for?

Reply:
For some reason, I thought the map was `NodeID -> qsetHash` (as it's what we use in some places), so we could feed the quorum set hashes to the hasher.
Not that, so current code is fine.